polyphemus/config.yml.template

```diff
@@ -14,7 +14,21 @@
     :user: developer
     :password: password
     :search_path: [ public ]
-
+  :auth_redirect: https://janus.development.local
```

Comment:
Note these changes to the `config.yml` will have to be propagated to your dev environments.

Reply:
And to Chef / the deployed roles...

Reply:
Rgr rgr.  I can take take care of adding this to roles.
Also, when you update the config.yml.template now, it will notify you the file has been updated and ask if you want to copy it over and replace the existing one, but in the future it could be a yml merge instead.